public/index.html

```diff
@@ -2,8 +2,15 @@
   <html>
     <head>
       <title>FEC Proxy</title>
+      <link rel="stylesheet" href="http://localhost:3002/style.css">
+      <link rel="stylesheet" href="http://localhost:3003/styles.css">
     </head>
     <body>
-      hello world
+      <div id='info-section'></div>
+      <div id='reviews'></div>
+      <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
```

Comment:
idk if this is needed since we are using jsx

Reply:
hmm i'll remove and test, will send another PR if working